github/tests/Issue606.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+# ########################## Copyrights and license ############################
+#                                                                              #
+# Copyright 2018 bbi-yggy <yossarian@blackbirdinteractive.com>                 #
+#                                                                              #
+# This file is part of PyGithub.                                               #
+# http://pygithub.github.io/PyGithub/v1/index.html                             #
+#                                                                              #
+# PyGithub is free software: you can redistribute it and/or modify it under    #
+# the terms of the GNU Lesser General Public License as published by the Free  #
+# Software Foundation, either version 3 of the License, or (at your option)    #
+# any later version.                                                           #
+#                                                                              #
+# PyGithub is distributed in the hope that it will be useful, but WITHOUT ANY  #
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS    #
+# FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more #
+# details.                                                                     #
+#                                                                              #
+# You should have received a copy of the GNU Lesser General Public License     #
+# along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
+#                                                                              #
+# ##############################################################################
+
+import Framework
+
+import github
+
+# TODO: test for retrieving organization projects
+# TODO: test for getting issue / pull request content from card
+
+# https://github.com/PyGithub/PyGithub/issues/606
+# https://github.com/PyGithub/PyGithub/pull/854
+class Issue606(Framework.BasicTestCase):
+    def setUp(self):
+        Framework.BasicTestCase.setUp(self)
+        self.g = github.Github(self.oauth_token)
```

Comment:
This line was the key to being able to record with `--auth_with_token` and replay without.